Pandemic/Board.cpp

```diff
@@ -206,6 +210,97 @@ void Board::setPlayerCardsFromLoad() {
 	cardManager->moveCardToDeck();
 }
 
+void Board::epidemicCardAction() {
+	incrementInfectionRate();
+	bottomInfectionCard = cardManager->getInfectionCardDeck()->at(cardManager->getInfectionCardDeck()->size() - 1);
+	string areaColor = bottomInfectionCard->getLocation().getArea();
+	int diseaseCubeCounter;
+
+	cout << "=============================================================================================================" << endl;
+	cout << "                                        EPIDEMIC CARD STARTED!!!" << endl;
+	cout << "=============================================================================================================" << endl;
+
+	if (areaColor == "Black" & !isBlackCured()) {
```

Comment:
For the code below, make sure to use the defined constants in the Board.h file. You'll notice for example #define BLACK "Black".